src/subgraph/subgraph.service.spec.ts

```
@@ -11,12 +11,13 @@
 import { Factory, getFactory } from '../../test/factory/factory';
 import { SubgraphModule } from './subgraph.module';
 import { SubgraphService } from './subgraph.service';
-import { UserEntity } from '../user/user.entity';
+import { SubgraphClientService } from '../subgraph-client/subgraph-client.service';
+import { SubgraphClientServiceMock } from '../../test/mocks/subgraph-client.mock';
 
 describe('SubgraphService tests', () => {
   let module: TestingModule;
   let service: SubgraphService;
-  let factory: Factory;
+  let _factory: Factory;
```

ESLint

[error] '_factory' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)